cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -373,6 +374,9 @@ public static final class AppMetadataStore {
     public static final String APPLICATION_FIELD = "application";
     public static final String VERSION_FIELD = "version";
     public static final String APPLICATION_DATA_FIELD = "application_data";
+    public static final String OWNER_FIELD = "owner";
```

Comment:
The new fields are discussed in LCM design doc: https://docs.google.com/document/d/1d4QZFGJqbrUb1kVkdB2OhD-WLJHKBcogdJ4_EzN9nXo/edit?resourcekey=0--Eq_ZhKnGHZxyagKwEBgZg&disco=AAAAeleNX78